S2VX.Game/Story/Note/S2VXNote.cs

```diff
@@ -1,29 +1,42 @@
-﻿using Microsoft.CodeAnalysis.CSharp.Syntax;
-using osu.Framework.Allocation;
+﻿using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
 using osu.Framework.Utils;
 using osuTK;
 using System;
 
 namespace S2VX.Game.Story.Note {
-    public abstract class S2VXNote : RelativeBox, IComparable<S2VXNote> {
+    public abstract class S2VXNote : CompositeDrawable, IComparable<S2VXNote> {
         public double EndTime { get; set; }
         public Vector2 Coordinates { get; set; } = Vector2.Zero;
 
         public Approach Approach { get; set; }
 
+        private RelativeBox Outer { get; } = new RelativeBox();
```

Comment:
Personally I'd like this variable name to be a bit more descriptive like "OuterBox" or "OuterSquare" for readability

Reply:
I'll make this change.

Reply:
I renamed these to BoxOuter and BoxInner. We sort of have a convention in other files to put the UI element type in front.